process/block/metablock.go

```diff
@@ -880,13 +900,17 @@ func (mp *metaProcessor) createAndProcessCrossMiniBlocksDstMe(
 
 		if maxTxSpaceRemained > 0 && maxMbSpaceRemained > 0 {
 			snapshot := mp.accountsDB[state.UserAccountsState].JournalLen()
-			currMBProcessed, currTxsAdded, hdrProcessFinished := mp.txCoordinator.CreateMbsAndProcessCrossShardTransactionsDstMe(
+			currMBProcessed, currTxsAdded, hdrProcessFinished, createErr := mp.txCoordinator.CreateMbsAndProcessCrossShardTransactionsDstMe(
```

Comment:
I think in case of createMbsAndProcessCrossShardTransactionDstMe there should not be possible to return nil. if there is a miniblock type cross shard which can't be processed, it should be ignored. there is no need to stop metachain if the shard created something with invalid miniblock type.

Reply:
fixed